src/main/resources/templates/course.html

```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
     <div layout:fragment="content">
-        <h1>Course</h1>
+        <h1>Course  <span th:text="${course.name}"></span> </h1>
```

Comment:
consider this line instead:
`<h1 th:text="${course.name}">course_name</h1>`
thymeleaf will replace 'course_name' with the value of course.name

Reply:
my bad, yeah that would have been better